.github/workflows/test.yml

```diff
@@ -0,0 +1,51 @@
+name: Test
+
+on:
+  push:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          sudo apt update
+          sudo apt install espeak git
+      - name: Upgrade pip
+        # so we can take advantage of pyproject.toml build-dependency support
+        run: python3 -m pip install --upgrade pip
+      - name: Install TTS
+        run: |
+          python3 -m pip install -e .
+          python3 setup.py egg_info
+      - name: Lint check
+        if: github.event_name == 'pull_request'
+        run: |
+          git remote set-branches --add origin ${GITHUB_REF}
+          git fetch
+          python3 -m pip install --quiet --upgrade cardboardlint pylint
+          cardboardlinter --refspec origin/${GITHUB_REF} -n auto
+      - name: Unit tests
+        run: |
+          nosetests tests --nocapture
+          ./tests/test_server_package.sh
+      - name: Test scripts
+        run: |
+          # test model training scripts
+          ./tests/test_tts_train.sh
```

Comment:
full list of test scripts 

```
./tests/test_tts_train.sh
./tests/test_glow-tts_train.sh 
./tests/test_vocoder_gan_train.sh 
./tests/test_vocoder_wavernn_train.sh 
./tests/test_vocoder_wavegrad_train.sh 
```

Reply:
There seem to be more than that:
```
test_glow-tts_train.sh
test_server_package.sh
test_tacotron_train.sh
test_tts_train.sh
test_vocoder_gan_train.sh
test_vocoder_wavegrad_train.sh
test_vocoder_wavernn_train.sh
```

Reply:
just ignore test_tts_train. It's supposed to be removed.

Reply:
and removed on the latest dev

Reply:
Rebased :+1: